### PR TITLE
Test against JSONTestSuite

### LINKED ISSUE #8: Test against JSONTestSuite
- https://github.com/nst/JSONTestSuite

### PR DESCRIPTION
**Pull request by fsvehla**
Closes #8 
Results as of 2020-09-24 in #55 

Should be useful when investigating those issues (remove `@ignore`)

### CODE REVIEW

**Comment by fsvehla** [comment]
@jdegoes What is the preferred method to give JSONTestSuite some credit?

**Comment by jdegoes** [comment]
In the source code and in the credits of the README, as well as reproducing the copyright in the appropriate file.

File: zio-json/jvm/src/test/scala/zio/json/JsonTestSuiteSpec.scala [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+import testzio.json.TestUtils._
+import zio._
+import zio.json._
+import zio.json.ast.Json
+import zio.test.Assertion._
+import zio.test.TestAspect._
+import zio.test._
+
+// zioJsonJVM/testOnly testzio.json.EncoderSpec
```
Comment (fommil) [inline zio-json/jvm/src/test/scala/zio/json/JsonTestSuiteSpec.scala]: out of date comment

**Review by jdegoes — APPROVED** [review]

**Comment by jdegoes** [comment]
Good to merge when conflicts are fixed and attribution is added.

**Pull request merged**